refactor(curriculum): updating step 22 of stats calculator project (#54312)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/661899113600ef307d8a7487.md

---
id: 661899113600ef307d8a7487
title: Step 23
challengeType: 0
dashedName: step-23
---

# --description--

> To get the median for an odd number of elements in an array, you will need to find the middle number.

Reviewer: ```suggestion
To get the median of an array with an odd number of elements, you will need to find and return the middle number.
```
